index.js

```diff
@@ -165,23 +210,9 @@ function parse (opts) {
 
   // pass in custom additional entries e.g. ['./test.js']
   if (opts.entries) {
-    if (typeof opts.entries === 'string') opts.entries = [opts.entries]
-
-    debug('globby resolving', opts.entries)
-
-    globby.sync(opts.entries, {
-      cwd: path.dirname(pkgPath),
-      absolute: true,
-      expandDirectories: false
-    }).forEach(entry => {
-      // Globby yields unix-style paths.
-      const normalized = path.resolve(entry)
-
-      if (paths.indexOf(normalized) === -1) {
-        debug('globby resolved', normalized)
-        paths.push(normalized)
-      }
-    })
+    paths = paths.concat(
```

Comment:
Can we put the path logic into another function so we can avoid condition paths altogether?